scripts/st2client-startup.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 
+counter=0
+
 # st2client startup and registration
-while true; do
+while [ "$counter" -lt 5 ]; do
   ACTIONS=$(st2 action list)
   if [ "$?" -ne 0 ]; then
-    echo "unable to reach downstream, will try again"
-    sleep 5
+    counter=$((counter+1))
+    echo "unable to reach downstream, will try again in $counter seconds..."
```

Comment:
Looks like we print `$counter` seconds here, while the actual sleep is `$counter * 5` below.